src/method-lookup.md

```diff
@@ -37,6 +37,7 @@ probe phase produces a "pick" (`probe::Pick`), which is designed to be
 cacheable across method-call sites. Therefore, it does not include
 inference variables or other information.
 
+[UFCS]: https://en.wikipedia.org/wiki/Uniform_Function_Call_Syntax
```

Comment:
Perhaps this might be a better link: https://github.com/rust-lang/rfcs/blob/master/text/0132-ufcs.md